opentreemap/treemap/lib/map_feature.py

```diff
@@ -303,8 +309,41 @@ def context_dict_for_map_feature(request, feature):
         'address_full': feature.address_full,
         'upload_photo_endpoint': None,
         'photos': None,
+        'share': None,
     }
 
     _add_eco_benefits_to_context_dict(instance, feature, context)
 
     return context
+
+
+def _add_share_context(context, request, photos):
+    if len(photos) > 0:
+        photo_url = photos[0].thumbnail.url
+    elif context.get('has_tree', False):
+        photo_url = settings.STATIC_URL + "img/tree.png"
+    else:
+        photo_url = settings.STATIC_URL + "img/otmLogo126.png"
```

Comment:
It's actually a generic tree when a tree is present and the OTM logo otherwise. Maybe 
```python
elif context.get('has_tree', False) == True:
```
would be clearer